documentation/release_notes.rst

```diff
@@ -8,26 +8,23 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
-New in 2021.7.0
+New in 2021.7.1
 ===============
 
-Deprecation Notice
-------------------
-- Deprecated support of C++11 for Parallel API with host execution policies (``seq``, ``unseq``, ``par``, ``par_unseq``).
-  C++17 is the minimal required version going forward.
+New Features
+------------
+- Added namespace alias ``dpl = oneapi::dpl``.
+- Added a function that constructs a zip_iterator using a std::tuple of iterators.
+- Added 9 more serial-based versions of algorithms: ``is_heap``, ``is_heap_until``, ``make_heap``, ``push_heap``, ``pop_heap``, ``is_sorted``, ``is_sorted_until``, ``partial_sort``, ``partial_sort_copy``. Please refer to `Tested Standard C++ API Reference https://github.com/oneapi-src/oneDPL/blob/main/documentation/library_guide/tested_standard_cpp_api.rst#tested-standard-c-api-reference`.
   
 Fixed Issues
```

Comment:
@MikeDvorskiy should your fix for in-place scan (https://github.com/oneapi-src/oneDPL/commit/7595b51cdee2044ae4cc9454bbcb1ef874c7dad3) be mentioned in release notes?

Reply:
Fix for in-place scan is still work in progress